script/utils/diamondEMERGENCYPauseGitHub.sh

```
@@ -116,6 +130,37 @@ function handleNetwork() {
+    "bsc-testnet" | "localanvil" | "sepolia" | "mumbai" | "lineatest")
+      echo "skipping $NETWORK (Testnet)"
+      return 0
+      ;;
+  esac
+
+  # get diamond address for this network
+  # DIAMOND_ADDRESS=$(getContractAddressFromDeploymentLogs "$NETWORK" "production" "LiFiDiamond")
+  DIAMOND_ADDRESS="0xD3b2b0aC0AFdd0d166a495f5E9fca4eCc715a782"  # TODO: remove <<<<<<<<<---------------------------------------------------------------------------------------- (STAGING DIAMOND ON POL, ARB, OPT)
+
+  # check if the diamond is paused by calling owner() function and analyzing the response
+  local RESPONSE=$(cast call "$DIAMOND_ADDRESS" "owner()" --rpc-url "$RPC_URL")
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)